.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+.jpb
```

Comment:
Why do we need it?

Reply:
because I use a plugin 'JPA Buddy' and it creates a package '.jpb'